src/ludamus/templates/chronology/_compact_schedule.html

```
@@ -50,7 +50,7 @@
         {# container — fitRail (event-timeline.ts) thins the markers until they #}
         {# fit the screen, and a non-scrollable rail chains wheel input to the #}
         {# page natively. #}
-        <nav class="schedule-rail sticky top-20 max-h-[calc(100dvh-6rem)] shrink-0 touch-none select-none self-start overflow-hidden py-1 text-foreground-muted"
+        <nav class="schedule-rail sticky top-20 max-h-[calc(var(--app-vh,100dvh)-6rem)] shrink-0 touch-none select-none self-start overflow-hidden py-1 text-foreground-muted"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)